mmv1/products/certificatemanager/DnsAuthorization.yaml

```diff
@@ -105,7 +105,7 @@ properties:
     immutable: true
     values:
       - :FIXED_RECORD
-      - :PER_RPOJECT_REOCRD
+      - :PER_PROJECT_RECORD
```

Comment:
Seems like when an ENUM field is marked with default_from_api, TF accepts whatever value that comes from the API without checking if this value is a valid ENUM value. It looks to me as a bug in TF side.

Reply:
not necessarily, since we know that the `type` value will be returned as any of the values listed, being either `FIXED_RECORD` or `PER_PROJECT_RECORD`